### PR TITLE
ref(tsc): usageChart utils to tsx

### PR DESCRIPTION
**Pull request by IanWoodard**
Convert utils.spec.jsx to tsx

### CODE REVIEW

File: static/app/views/organizationStats/usageChart/utils.spec.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ describe('getDateFromMoment', () => {
   // Ensure date remains in UTC
   it('shows the date if interval is >= 24h', () => {
     expect(getDateFromMoment(start)).toBe('Jul 9');
-    expect(getDateFromMoment(start, '7d')).toBe('Jul 9');
+    expect(getDateFromMoment(start, '2d')).toBe('Jul 9');
```
Comment (cathteng) [inline static/app/views/organizationStats/usageChart/utils.spec.tsx:16]:
```suggestion
    expect(getDateFromMoment(start, '7d')).toBe('Jul 9');
```
this should be kept the same, right?

Comment (IanWoodard) [reply static/app/views/organizationStats/usageChart/utils.spec.tsx:16]: I changed it since 7d is not a valid IntervalPeriod.

Comment (IanWoodard) [reply static/app/views/organizationStats/usageChart/utils.spec.tsx:16]: That is Cathy's suggested change. I changed it from `7d` to `2d`.

Comment (EricHasegawa) [reply static/app/views/organizationStats/usageChart/utils.spec.tsx:16]: mb 👍

Comment (cathteng) [reply static/app/views/organizationStats/usageChart/utils.spec.tsx:16]: oh i see there's an `IntervalPeriod` object

**Review by cathteng — APPROVED** [review]

**Pull request merged**